mijia-homie.service

```diff
@@ -10,6 +13,7 @@ Environment=RUST_BACKTRACE=1
 Environment=RUST_LIB_BACKTRACE=1
 ExecStart=/home/pi/mijia-homie
 Restart=always
+RestartSec=1
```

Comment:
I think this should be longer (at least 5 seconds, perhaps 10 seconds or more) to avoid getting everything bogged down with continuous restarts when it keeps failing.